cli/.goreleaser.yaml

```diff
@@ -32,7 +35,10 @@ dockers:
       - "ghcr.io/cloudquery/cloudquery:latest-linux-arm64"
       - "ghcr.io/cloudquery/cloudquery:{{.Version}}-linux-arm64"
       - "ghcr.io/cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}-linux-arm64"
+    use: buildx
     build_flag_templates:
+      - "--pull"
```

Comment:
Does buildx not account for the platform when checking if the image is already in the cache, or what is the reason for adding `--pull` ?

Reply:
I've added it as per Erez's [instruction](https://github.com/cloudquery/cloudquery/issues/16218#issuecomment-1903804664) feel free to take over this issue if you're comfortable with this as I'm really not.

Reply:
Oh I see, it's from the goreleaser example. I guess it doesn't hurt, we just won't use the cache. I also don't know whether that's necessary 😅

Reply:
I think the reason to add `--pull` is to refresh any base images our image depends on. For example if we depend on `v2` tag of an image we'll pull the latest changes to that tag.

Reply:
Ah I see, makes sense yeah.